cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -607,30 +605,27 @@ private void moveReplicas() {
         waitForExecutionTaskToFinish();
         inExecutionTasks = _executionTaskManager.inExecutionTasks();
       }
-      if (_executionTaskManager.inProgressTasks().isEmpty()) {
-        LOG.info("Partition movements finished.");
-      } else if (_stopRequested.get()) {
-        ExecutionTaskManager.ExecutionTasksSummary executionTasksSummary = _executionTaskManager.getExecutionTasksSummary();
-        LOG.info("Partition movements stopped. {} in-progress, {} pending, {} aborting, {} aborted, {} dead, "
-                 + "{} remaining data to move.",
-                 executionTasksSummary.inProgressTasks().size(),
-                 executionTasksSummary.remainingPartitionMovements().size(),
-                 executionTasksSummary.abortingTasks(),
-                 executionTasksSummary.abortedTasks().size(),
-                 executionTasksSummary.deadTasks().size(),
-                 executionTasksSummary.remainingDataToMoveInMB());
-      }
+      ExecutionTasksSummary executionTasksSummary = _executionTaskManager.getExecutionTasksSummary(Collections.emptyList(), false);
+      LOG.info("Partition movements {} with {} tasks {}, {} tasks in-progress, {} tasks aborting, {} tasks aborted, {} tasks dead, {} tasks completed.",
+               _stopRequested.get() ? STOPPED : FINISHED,
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.PENDING),
+               _stopRequested.get() ? CANCELLED : PENDING,
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.IN_PROGRESS),
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.ABORTING),
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.ABORTED),
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.DEAD),
+               executionTasksSummary.taskStat().get(REPLICA_ACTION).get(ExecutionTask.State.COMPLETED));
```

Comment:
Is it possible to avoid calling `executionTasksSummary.taskStat().get(REPLICA_ACTION)` repetitively?